package.json

```diff
@@ -41,6 +41,7 @@
   },
   "scripts": {
     "test": "mocha",
+    "debugtest": "mocha --debug-brk $npm_package_options_mocha",
```

Comment:
Yeah I think it just clutters the package.json file

You can run with debug flags manually with

`npm test -- --debug-brk`